src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderTrustAccount.java

```diff
@@ -31,7 +31,6 @@ public class OffenderTrustAccount {
     @Column(name = "OFFENDER_ID", nullable = false, insertable = false, updatable = false)
     private Long offenderId;
 
-    @Id
```

Comment:
Not actually included in the id and causes an error in latest Hibernate verison.